### DISCUSSION TITLE
mode auto : pas d'encart indiquant la nécessité d'activer les cookies recaptcha V2 ou V3

### DISCUSSION THREAD

**Discussion by plaste** [AmauriC/tarteaucitron.js#1024] [Q&A] [answered]
Bonjour,
Nous utilisons recaptcha V2 sur un formulaire de contact CF7 Wordpress.
Nous utilisons tarteaucitron pro en mode automatique. 

Si le visiteur refuse les cookies, il n'y a aucune indication sur le formulaire que recaptcha est refusé. Le visiteur ne peut donc pas valider le formulaire mais sans savoir pourquoi.
L'idéal serait qu'un petit encart apparaisse au niveau du recaptcha pour indiquer qu'il est nécessaire d'autoriser les cookies pour que le module s'affiche (à l'image de l'encart youtube ou vimeo qui s'affiche à la place de la vidéo dans ce cas là).

Le même phénomène se produit aussi sur un formulaire d'inscription pour être informé du retour en stock d'un produit (donc hors de CF7), et là nous ne pouvons même pas prévoir un petit message d'erreur pour indiquer au visiteur d'autoriser le service reCaptcha. L'encart serait donc indispensable. Nous avons donc du désactiver recaptcha à cet endroit.

Voici la page de formulaire CF7 où vous pouvez constater le soucis : https://les-woodcutters.fr/contact/

Il me semble qu'avant le module affichait bien le message pour recaptcha... mais je peux me tromper.
Vous est-il possible de faire quelque chose ?

Merci d'avance pour la prise en compte de cette demande.

**Comment by AmauriC** [comment, marked as answer]
Sur https://les-woodcutters.fr/contact/ il y a bien le bouton pour activer recaptcha mais en bas de page.

tarteaucitron ajoute ce bouton à la place de la balise `<script>`, si la balise est dans le footer, le bouton sera à cette endroit.
C'est souvent les système d'optimisation js qui déplacent les balises en fin de DOM.

**Reply by plaste** [reply]
Merci Amauri. L'encart n'apparaissait pas toute à l'heure mais c'est le cache qui ne devait pas encore l'avoir traité !
Il y a effectivement un WP-ROCKET qui tourne sur ce site.
Désolé pour cette question qui était du coup sans objet et merci pour votre retour !
